engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-details.html

```diff
@@ -69,23 +69,11 @@
         </a>
       </li>
       <li uib-dropdown
-          ng-class="{active: stateIncludes('content-host.subscriptions.list') || stateIncludes('content-host.subscriptions.add') || stateIncludes('content-host.events')}">
-        <a uib-dropdown-toggle>
-          <span translate>Subscriptions</span>
-          <i class="fa fa-chevron-down"></i>
+          ng-class="{active: stateIncludes('content-host.subscriptions.list') || stateIncludes('content-host.subscriptions.add')}">
+        <a translate
+           ui-sref="content-host.subscriptions.list">
+          Subscriptions
         </a>
-        <ul uib-dropdown-menu>
-          <li>
-            <a ui-sref="content-host.subscriptions.list" translate>
-              Subscriptions
-            </a>
-          </li>
-          <li>
-            <a ui-sref="content-host.events" translate>
-              Events
```

Comment:
I think there should be another angular template for the actual contents of the Events tab which can be removed

Reply:
Oh yes. I forgot to remove that. I'm on it.